modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/impl/MetaStorageRaftGroupEventsListener.java

```diff
@@ -128,6 +146,10 @@ private interface OnLeaderAction {
         CompletableFuture<Void> apply(MetaStorageServiceImpl service, long term);
     }
 
+    private interface OnStatusAction {
```

Comment:
Missing `FunctionalInterface`.
Can get rid of the `OnLeaderAction`?

Reply:
Leader action is useful. And anyway, this will all change with replica based ms